time/time.go

```diff
@@ -27,7 +28,7 @@ func (t *Time) MarshalJSON() ([]byte, error) {
 // The time is expected to be in Deis' datetime format.
 func (t *Time) UnmarshalText(data []byte) (err error) {
 	tt, err := time.Parse(DeisDatetimeFormat, string(data))
-	if err != nil {
+	if _, ok := err.(*time.ParseError); ok {
 		tt, err = time.Parse(PyOpenSSLTimeDateTimeFormat, string(data))
 	}
```

Comment:
what about other errors? I'd like to be defensive here in case other errors ever are returned